Frontend/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
Do we need this file?

Reply:
I thought so, but I removed it and nothing happened. Hopefully not.